### PR TITLE
Update 04-palette-virtual-clusters.md

### PR DESCRIPTION
**Pull request by ritawatson**
This PR fixes a link to a renamed doc and updates an example to remove 'sandbox' terminology.

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1038#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 656ee8beb856b518ff9ac527158544220bb5024f | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1038/librarium_build/1602820566803288064) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/04-clusters/04-palette-virtual-clusters.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Palette provisions and orchestrates Palette Virtual Clusters and makes it easy t
 
 Palette also supports Day 2 operations such as upgrades, backup/restore and more, to keep Palette Virtual Clusters secure, compliant, and up to date. Additionally, it provides visibility into the workloads running inside your Palette Virtual Clusters and its associated costs.
 
-To get started and create your Palette Virtual Cluster, see the [Palette Virtual Cluster Quick Start](/clusters/palette-virtual-clusters/virtual-cluster-quickstart) page.
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/04-palette-virtual-clusters.md:26]: @ritawatson, if you are changing a file name, then that requires a redirect. If we can avoid changing a file name, then we should.

File: content/docs/04-clusters/04-palette-virtual-clusters.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ The NGINX Ingress Controller must be deployed on the Host cluster with SSL passt
 
    For example:
 
-   `*.sandbox.host.1.spectrocloud.com`
+   `*.starship.te.spectrocloud.com`
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/04-palette-virtual-clusters.md:44]: I was told this value is not changing.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]

File: content/docs/04-clusters/04-palette-virtual-clusters.md (hunk 1 above)
Comment (ritawatson) [reply content/docs/04-clusters/04-palette-virtual-clusters.md:26]: @karl-cardenas-coding I didn't know that. Could you help me create a redirect?

Comment (karl-cardenas-coding) [reply content/docs/04-clusters/04-palette-virtual-clusters.md]: Resolved

**Review by karl-cardenas-coding — APPROVED** [review]
LGTM 😎

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1038#pullrequestreview-1223638556" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1038#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**